src/android/FileHelper.java

```diff
@@ -97,10 +98,19 @@ public static String getRealPathFromURI_API11_And_Above(final Context context, f
             else if (isDownloadsDocument(uri)) {
 
                 final String id = DocumentsContract.getDocumentId(uri);
-                final Uri contentUri = ContentUris.withAppendedId(
-                        Uri.parse("content://downloads/public_downloads"), Long.valueOf(id));
-
-                return getDataColumn(context, contentUri, null, null);
+                if (!TextUtils.isEmpty(id)) {
```

Comment:
Instead of using TextUtils just for checking if is empty, use id != null && id.length() > 0